CMakeExt/HDF5.cmake

```diff
@@ -61,10 +53,23 @@ find_package_handle_standard_args(
   HDF5_INCLUDE_DIRS
 )
 
+
 # set flags
 if(HDF5_FOUND)
-#  set (HDF5_LINKER_FLAGS "-lhdf5_hl -lhdf5 -ldl -lm -lz")
-   set (HDF5_LINKER_FLAGS $ENV{HDF5_LIB} ${SZIP_LIB} -lz)
+  CHECK_SYMBOL_EXISTS(
+    H5_HAVE_PARALLEL
+    "${HDF5_INCLUDE_DIRS}/H5pubconf.h"
```

Comment:
I'll try that in the evening. The errors are strange, but it works nevertheless. Seems like a problem with static linking as the directory is chosen instead of the library (xx.a).